src/App.js

```diff
@@ -1,29 +1,9 @@
-import logo from './logo.svg';
 import './App.css';
+import Calculator from './components/Calculator';
 
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit
-          {' '}
-          <code>src/App.js</code>
-          {' '}
-          and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
-}
-
+const App = () => (
+  <div>
+    <Calculator />
+  </div>
+);
```

Comment:
For this project, you are required to make your components classes class-based components.

- [ ] Kindly make this component a class-based react component. The goal is for you to know both ways of creating react apps with the good old class-based and later you'll refactor all your components into functional hooks as your milestone. 👍 Kindly work on the change

Reply:
Well resolved @petushka1 👍🏽